import-automation/executor/README.md

```diff
@@ -0,0 +1,153 @@
+which runs user scripts, generates CSVs and MCFs, and import them to the Data
+Commons knowledge graph using the importer.
+
+## User Manual
+
+### Specifying Import Targets
+Import targets are specified in the commit message using the IMPORTS tag.
+The system accepts one or two of the following formats depending on the files
+affected by the commit.
+
+1. Absolute import name:
+   {path to the directory containing the manifest file}:{import name in the manifest}
+2. Relative import name: {import name in the manifest}
+
+In the commit message, use IMPORTS={comma separated list of import names
+without spaces between the elements} to specify import targets.
+The commit message does not have to only contain the tag and list.
+
+A commit can modify files in multiple directories that contain manifest files.
+The system will detect the affected directories based on paths of changed files.
+If only one directory is affected, both absolute and relative import names
+are accepted. If multiple directories are affected, absolute import names must
+be used. You can also use IMPORTS=all to ask the system to run all affected
+imports and use IMPORTS={path to the directory containing the manifest file}:all
+to run all imports in that directory, but they are discouraged as they are not
+explicit.
+
+
+### Example Commit Messages
+
+Assuming the following directory structure:
+```
+.
+└── data
+    └── scripts
+        └── us_bls
+            ├── cpi
+            │   ├── README.md
+            │   ├── c_cpi_u_1999_2020.csv
+            │   ├── c_cpi_u_1999_2020.mcf
+            │   ├── c_cpi_u_1999_2020.tmcf
+            │   ├── c_cpi_u_1999_2020_import_config.txt
+            │   ├── cpi_u_1913_2020.csv
+            │   ├── cpi_u_1913_2020.tmcf
+            │   ├── cpi_u_1913_2020_import_config.txt
+            │   ├── cpi_w_1913_2020.csv
+            │   ├── cpi_w_1913_2020.mcf
+            │   ├── cpi_w_1913_2020.tmcf
+            │   ├── cpi_w_1913_2020_import_config.txt
+            │   ├── generate_csv.py
+            │   └── manifest.json
+            └── jolts
+                ├── BLSJolts.csv
+                ├── BLSJolts.tmcf
+                ├── BLSJolts_StatisticalVariables.mcf
+                ├── README.md
+                ├── __init__.py
+                ├── bls_jolts.py
+                ├── import.config
+                └── manifest.json
+```
+
+Assuming scripts/us_bls/cpi/manifest.json has
+```json
+{
+    "import_specifications": [
+        {
+            "import_name": "USBLS_CPIAllItemsAverage",
+            ...
+        }
+    ]
+}
+```
+
+Assuming scripts/us_bls/jolts/manifest.json has
+```json
+{
+    "import_specifications": [
+        {
+            "import_name": "BLS_JOLTS",
+            ...
+        }
+    ]
+}
+```
+
+If the commit only changes files in scripts/us_bls/cpi:
+- To import USBLS_CPIAllItemsAverage:
+  - "fix syntax error IMPORTS=scripts/us_bls/cpi:USBLS_CPIAllItemsAverage"
```

Comment:
```suggestion
  - "fix syntax error IMPORTS=scripts/us_bls/cpi:USBLS_CPIAllItemsAverage" OR
```

I am not sure if "OR" is the best way to express what I am trying to convey.  It may not be clear to a reader that, when multiple options are shown, choose only one.

Reply:
fixed